R/create_output_type_item.R

```diff
@@ -190,9 +190,14 @@ create_output_type_point <- function(output_type = c("mean", "median"),
 #'   value_type = "double"
 #' )
 #' create_output_type_sample(
-#'   required = 1:10, optional = 11:15,
-#'   value_type = "double"
-#' )
+#'  is_required = TRUE,
+#'  output_type_id_type = "integer",
+#'  min_samples_per_task = 70L, max_samples_per_task = 100L,
+#'  value_type = "double",
+#'  value_minimum = 0L,
+#'  value_maximum = 1L,
+#'  branch = "br-v3.0.0"
+#'  )
```

Comment:
Minor suggestion here to match indentation convention used in the example just above: 2 spaces indentation relative to a single-space buffer next to `#'`.
```suggestion
#'   is_required = TRUE,
#'   output_type_id_type = "integer",
#'   min_samples_per_task = 70L, max_samples_per_task = 100L,
#'   value_type = "double",
#'   value_minimum = 0L,
#'   value_maximum = 1L,
#'   branch = "br-v3.0.0"
#' )
```